packages/ui/DropDown/index.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  useFloating,
+  useClick,
+  useInteractions,
+  autoUpdate,
+  offset,
+  flip,
+  shift,
+  useDismiss,
+} from '@floating-ui/react-dom-interactions';
+
+interface DropDown {
+  parent: JSX.Element;
+  children: JSX.Element;
+  open: boolean;
+  setOpen: any;
```

Comment:
pass an `onClose` instead and type it correctly.

Reply:
done